hindsight/ConnectFlow/UI/LoginViewController.swift

```diff
@@ -14,17 +14,37 @@ import RxSwift
 
 class LoginViewController: UIViewController {
 
-    lazy var facebookConnect: UIButton = {
-        let button = UIButton()
-        button.setTitle("Connect With Facebook", for: .normal)
-        button.backgroundColor = ColorName.hindsightBlue.color
-        return button
-    }()
-
     let viewModel: LoginViewModelProtocol
 
     let disposeBag = DisposeBag()
 
+    lazy var logoImageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var backgroundImageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var facebookConnect: HindsightLoginButton = {
+        let normalState = LoginButtonState(text: "Login With Facebook",
```

Comment:
Text should be put in `Localizable.strings`?

Reply:
I have pushed the new code to use generated Constants file from SwiftGen.